frontend/components/card/discussion/CardDiscussionInput.vue

```diff
@@ -83,27 +83,41 @@
           />
         </div>
       </div>
-      <div class="w-full md:w-full">
+      <div v-if="discussionInput.highRisk" class="w-full md:w-full">
         <textarea
           id="message"
           rows="4"
-          class="block p-2.5 w-full text-sm text-light-text bg-light-content rounded-lg border border-light-section-div dark:bg-dark-content dark:border-dark-section-div placeholder-light-special-text dark:placeholder-dark-special-text dark:text-dark-text focus-brand"
-          :placeholder="$t('components.card-discussion-input.leave-comment')"
+          class="block p-2.5 w-full text-sm text-light-text rounded-lg border border-red-500 placeholder-light-special-text focus-brand  dark:bg-dark-distinct font-bold red-text"
+          :placeholder="$t('components.card-discussion-input.leave-comment-highRisk')"
         ></textarea>
       </div>
+       <div v-else class="w-full md:w-full">
+        <textarea
+              id="message"
+              rows="4"
+              class="block p-2.5 w-full text-sm text-light-text rounded-lg border border-light-section-div placeholder-light-special-text dark:placeholder-dark-special-text dark:text-dark-text  dark:bg-dark-distinct focus-brand"
+              :placeholder="$t('components.card-discussion-input.leave-comment')"
+        ></textarea>
+    </div>
       <div class="flex items-center justify-between px-1">
         <p class="inline-flex items-center">
           {{ $t("components.card-discussion-input.markdown-support") }}
           <Icon class="mx-1" name="bi:markdown" size="1.25em"></Icon>
         </p>
-        <div class="flex space-x-1">
+        <div class="flex space-x-3 items-center">
+          <div v-if="discussionInput.highRisk" class="w-full md:w-full">
+            <div class="cursor-pointer rounded-lg p-1 text-light-text dark:text-light-action-red dark:bg-dark-action-red/20 dark:border-dark-action-red bg-light-action-red border border-light-text focus-brand" style="width: 3.3em; height: 2.4em; display: flex; justify-content: center; align-items: center;">
```

Comment:
Nice :) This is where we'd like a tooltip that you all would be welcome to work on!

Reply:
In another PR :)

Reply:
What do you recommend the tooltip comment to be 🤔

Reply:
We can do a different issue for this, @mahasultan :) I can write that up and ping you and the others in it to see if any of you have interest. For that issue though, maybe something like:

```
Please use usernames and do not write any personal information including:
- Names
- Home addresses
- Phone numbers
- Email addresses
```

We can discuss in the new issue though 😊

Reply:
We would love to work on this as soon as possible for our stretch task. please ping us !

Reply:
Making the issue now :)

Reply:
See #588 :)